UT-Earthworks-12d-1/readme.md

```diff
@@ -0,0 +1,29 @@
+
+### Intent
+
+IFC 4x3 RC2
+This scenario has an earthworks fill object.
+
+The Ifc file was generated by 12d Model. 
```

Comment:
```suggestion
The IFC file was generated by 12d Model. 
```

Reply:
Fixed when added the RC3 example in 6717bf2